data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,307 @@ mission "Coalition License Patch"
+	passengers 6
+	cargo "Yottrite Presentation Materials" 6.14
+	to offer
+		has "Coalition Yottrite 8: done"
+	source "Pelubta Station"
+	destination "Ring of Wisdom"
+	on offer
+		conversation
+			`You meet Lugglop in the spaceport and follow him to meet with Nasilor, Alituri, and Pichiie, who are overseeing some workers place a few sealed metal crates inside a container and the yottrite sample you just brought here into another.`
+			`	They detail how they have prepared all the material they needed and now just need a transport to <destination>.`
+			`	"If with us, you come, Captain, perhaps a better detailed description of the systems where the yottrite was found, you could provide," Nasilor says.`
+			`	"<payment>, I would pay you," Lugglop says.`
+			choice
+				`	"Of course I'll take you there. Have the containers loaded into my ship."`
+					goto accept
+				`	"What discoveries have you made about it?"`
+					goto findings
+			label findings
+			`	"To share our findings with you, allowed we are not," Pichiie says.`
+			`	"Forgive us you must, Captain, but told to keep everything a secret, we were," Lugglop continues. "Perhaps after our presentation, public our findings could be made."`
+			label accept
+			`	They all follow you to your ship after the containers are ready to be loaded.`
+			`	After they are put into your cargo, Nasilor, Alituri and Pichiie head to their bunks, but as Lugglop is coming on board, you see that two Heliarch agents, both Kimek, are accompanying him.`
+			`	"Here to ensure the secrecy of our discoveries, they are," he says. Seeing as they are only two more passengers, you agree to carry them to <planet>, and they all board your ship.`
+				accept
+	on complete
+		payment 310000
+		conversation
+			`	Immediately when you land, Lugglop gives you your payment of <payment>. You and the four researchers then follow some Heliarch agents that were waiting for you at the docks. You five are brought to an area akin to a theater with some type of podium on the stage. The seats are all taken by Heliarch agents.`
+			`	Lugglop's team apparently will wait until the contents of the two containers are brought here, so you're asked to describe the systems with supernovae where you encountered yottrite asteroids. This only lasts a few minutes, as the audience doesn't seem too keen on inquiring about details, though one particular Saryd did puzzle you when they asked if the all the supernovas you visited "were natural occurrences, or artificially produced."`
+			`	After you finish, and right as the materials for Lugglop's presentation arrives, you're escorted back to your ship by some agents. Several hours later, you're contacted by Lugglop, who says the prpesentation is finished, and his group is coming back to your ship. When they arrive, they each thank you for providing the means to perform their experiments. "Officially commended for our efforts, we were!" He says.` 
```

Comment:
```suggestion
			`	After you finish, and right as the materials for Lugglop's presentation arrives, you're escorted back to your ship by some agents. Several hours later, you're contacted by Lugglop, who says the presentation is finished, and his group is coming back to your ship. When they arrive, they each thank you for providing the means to perform their experiments. "Officially commended for our efforts, we were!" he says.` 
```
(I think) since this would normally be a comma and you're just using an exclamation mark to clarify his tone, the punctuation should still be treated as if it were a comma (if that makes any sense).